jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/NoContentExceptionMapper.java

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-package com.palantir.remoting1.servers;
+package com.palantir.remoting1.servers.jersey;
 
 import javax.ws.rs.core.NoContentException;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.ext.ExceptionMapper;
 import javax.ws.rs.ext.Provider;
 
 @Provider
-final class NoContentExceptionMapper implements ExceptionMapper<NoContentException> {
+public final class NoContentExceptionMapper implements ExceptionMapper<NoContentException> {
```

Comment:
it's used in the dropwizards tests. replaced by DropwizardServers.configure, revert public change.